OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -390,6 +391,7 @@ public void ResolveOrder(Actor self, Order order)
 					var cost = valued != null ? valued.Cost : 0;
 					var time = GetBuildTime(unit, bi);
 					var amountToBuild = Math.Min(fromLimit, order.ExtraData);
+					var hasPriority = order.OrderString == "StartPriorityProduction" ? true : false;
```

Comment:
The `? true : false` here isn't necessary as the `==`'s result is a `bool` already.

```suggestion
					var hasPriority = order.OrderString == "StartPriorityProduction";
```